### ISSUE TITLE
function-calc-no-invalid reads sass variables

### ISSUE THREAD

**Issue by clp-amsiq** [stylelint/stylelint#3924]
**function-calc-no-invalid**
Just updated my stylelint to 9.10.1 and added this new cool rule.
After adding it my lintin went loco. It seems that all places i have used a sass variable in a calc() it expects me to put space on each side of the "-"

```css
div {
  width: calc(100% - #{$container-padding});
}
```

Stylelint prints the error regarding the "-" in $container-padding
```
Expected space before "-" operator
Expected space after "-" operator
```

**Comment by evilebottnawi** [comment]
@clp-amsiq Duplicate https://github.com/stylelint/stylelint/issues/3908?

**Comment by hudochenkov** [comment]
Duplicates https://github.com/stylelint/stylelint/issues/3908

**Comment by clp-amsiq** [comment]
Yup duplicate. Sorry fo the inconvenience ;)
